storybookjs/storybook
# (#1513) Replace `console` with `client-logger` and `node-logger` packages

Issue: #1513

## What I did

Added a logger package containing a logger to be use in the browser and another to be used in node.

Note: This is for work in progress. Any feedback is more than welcome.

## How to test

Is this testable with jest or storyshots?
Jest

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
Yes, it will need to be documented for add-on developers

If your answer is yes to any of these, please make sure to include it in your PR.

## Screenshots

### Node
<img width="163" alt="screen shot 2017-11-23 at 11 24 28 am" src="https://user-images.githubusercontent.com/4509692/33182194-f364d672-d040-11e7-8cb7-053b680551ec.png">

### Browser

<img width="498" alt="screen shot 2017-11-24 at 10 24 27 pm" src="https://user-images.githubusercontent.com/4509692/33227113-14487658-d16a-11e7-8af7-8a8440a1874a.png">
